packages/asset-swapper/src/utils/market_operation_utils/types.ts

```diff
@@ -34,6 +35,7 @@ export enum ERC20BridgeSource {
     CurveUsdcDaiUsdtTusd = 'Curve_USDC_DAI_USDT_TUSD',
     LiquidityProvider = 'LiquidityProvider',
     CurveUsdcDaiUsdtBusd = 'Curve_USDC_DAI_USDT_BUSD',
+    Rfqt = 'Rfqt',
```

Comment:
Is this used anywhere?

Reply:
No, it's not.  It's a remnant from the first draft.  I've removed it here: https://github.com/0xProject/0x-monorepo/pull/2555/commits/4cc9ceabd21edecc28a430973d90f7ad96c0a2a0